src/index.ts

```diff
@@ -45,10 +46,18 @@ const run = async () => {
 	summary
 		.addRaw(`<em>Generated in workflow <a href=${getWorkflowSummaryURL()}>#${github.context.runNumber}</a></em>`)
 
+	let processedPrNumber: number | undefined = Number(prNumber);
+	core.info(`Received pull-request number: ${prNumber}`);
+
+	if (!Number.isSafeInteger(processedPrNumber) || processedPrNumber <= 0) {
+		processedPrNumber = undefined;
+	}
```

Comment:
Could you put this entire check into the `src/inputs/option.ts`? 

It feels like input validation should happen there.
Apart from that, very nice check about the `safeInterger` here 👍🏻

Reply:
Sure, the validation has now been moved to `option.ts`.